gatsby-node.js

```diff
@@ -75,136 +76,137 @@ const createCareerPages = async ({ actions }) => {
+          const { jobDescription } = position.jobDescriptions;
+          return Array.isArray(jobDescription);
+        };
 
-      const getSlug = (position) => {
-        if (position.satellytesPath) return position.satellytesPath;
-        const slug = position.jobDescriptions.jobDescription.find(
-          (description) =>
-            description.name.trim() === 'Slug' && description.value.trim(),
-        );
-        return `/career/${slug.value.trim()}/`;
-      };
+        const getSlug = (position) => {
+          if (position.satellytesPath) return position.satellytesPath;
+          const slug = position.jobDescriptions.jobDescription.find(
+            (description) =>
+              description.name.trim() === SLUG && description.value.trim(),
+          );
+          return `/career/${slug.value.trim()}/`;
+        };
 
-      positions.forEach((position) => {
-        if (!isValidPosition(position)) {
-          return null;
-        }
+        positions.forEach((position) => {
+          if (!isValidPosition(position)) {
+            return null;
+          }
 
-        // we need to normalize the jobDescription to an array, otherwise gatsby
-        // complains about different types for the same variable
-        position.jobDescriptions.jobDescription = Array.isArray(
-          position.jobDescriptions.jobDescription,
-        )
-          ? position.jobDescriptions.jobDescription
-          : [position.jobDescriptions.jobDescription];
-
-        position.jobDescriptions.jobDescription =
-          position.jobDescriptions.jobDescription
-            .map((description) => {
-              if (description.name.trim() === 'Slug') {
-                position.satellytesPath = `/career/${description.value.trim()}/`;
-                return undefined;
-              } else {
-                return {
-                  name: description.name.trim(),
-                  value: description.value.trim(),
-                };
-              }
-            })
-            .filter((position) => position);
-
-        const description = position.jobDescriptions.jobDescription.find(
-          (description) => description.name === PERSONIO_SHORT_DESCRIPTION_NAME,
-        );
-        if (!description || !description.value) {
-          console.warn(
-            `No description for job "${position.name}" (${position.id}) found!`,
+          // we need to normalize the jobDescription to an array, otherwise gatsby
+          // complains about different types for the same variable
+          position.jobDescriptions.jobDescription = Array.isArray(
+            position.jobDescriptions.jobDescription,
+          )
+            ? position.jobDescriptions.jobDescription
+            : [position.jobDescriptions.jobDescription];
+
+          position.jobDescriptions.jobDescription =
+            position.jobDescriptions.jobDescription
+              .map((description) => {
+                if (description.name.trim() === SLUG) {
```

Comment:
Why do we check here for the `SLUG` and in `getSlug()`. Can we reuse the `getSlug()`function here?